lib/step_flow/configuration.ex

```diff
@@ -51,4 +51,8 @@ defmodule StepFlow.Configuration do
       _ -> true
     end
   end
+
+  def metrics_enabled? do
+    get_var_value(StepFlow, :enable_metrics, false)
```

Comment:
as it can also get from env, the returned value can be `true`, `false` or a string ...
To prevent that, it requires to match on the string to assign the boolean status associated.

Reply:
If it s a string, i set it to false?